openmetadata-service/src/main/java/org/openmetadata/service/util/jdbi/DatabaseAuthenticationProviderFactory.java

```diff
@@ -35,7 +35,7 @@ private static Map<String, String> parseQueryParams(String jdbcURL) {
       URI uri = new URI(jdbcURL.substring(jdbcURL.indexOf(":") + 1));
       Map<String, String> queryPairs = new LinkedHashMap<>();
       String query = uri.getQuery();
-      if (query != null) {
```

Comment:
@plutaniano thanks for the PR. We actually have `nullOrEmpty` from `CommonUtils` you can use

Reply:
Done